test/WebJobs.Script.Tests.Integration/Host/WebJobsScriptHostServiceTests.cs

```diff
@@ -288,7 +288,7 @@ public async Task Restarts_CanCancel_Restarts()
             _testHost.Dispose();
             int buildCalls = 0;
 
-            var testHost = new TestFunctionHost(TestScriptPath, TestLogPath,
+            using (var testHost = new TestFunctionHost(TestScriptPath, TestLogPath,
```

Comment:
The tests in here weren't disposing of their hosts, which meant they'd run forever in the background. `Validate_HostLogs` validates the "host lock lease" message -- which would never fire because one of these hosts was hanging onto it.